api/v1alpha1/kafkauser_types.go

```diff
@@ -15,17 +15,28 @@
 package v1alpha1
 
 import (
+	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/banzaicloud/kafka-operator/api/v1beta1"
 )
 
 // KafkaUserSpec defines the desired state of KafkaUser
 // +k8s:openapi-gen=true
 type KafkaUserSpec struct {
-	SecretName  string           `json:"secretName"`
-	ClusterRef  ClusterReference `json:"clusterRef"`
-	DNSNames    []string         `json:"dnsNames,omitempty"`
-	TopicGrants []UserTopicGrant `json:"topicGrants,omitempty"`
-	IncludeJKS  bool             `json:"includeJKS,omitempty"`
+	SecretName     string           `json:"secretName"`
+	ClusterRef     ClusterReference `json:"clusterRef"`
+	DNSNames       []string         `json:"dnsNames,omitempty"`
+	TopicGrants    []UserTopicGrant `json:"topicGrants,omitempty"`
+	IncludeJKS     bool             `json:"includeJKS,omitempty"`
+	CreateCert     *bool            `json:"createCert,omitempty"`
```

Comment:
Wouldn't `bool` type just work fine instead of using a pointer here? If this field is omitted in the payload that it would be initialised with default value which is `false`